src/api/market.ts

```diff
@@ -33,3 +34,22 @@ export const getProduct = async (
   );
   return response.data;
 };
+
+export const postProduct = async (
+  token: string,
+  formData: ProductFormValues,
+  categoryId: number | undefined,
+) => {
+  if (categoryId === 0) {
+    throw new Error('카테고리를 선택해주세요.');
+  }
```

Comment:
categoryId 는 undefined 일 수도 있으니까 아래가 좀 더 general 한 error handling 일듯!

```suggestion
  if (!categoryId) {
    throw new Error('카테고리를 선택해주세요.');
  }
```

Reply:
falsy 값 더 공부해오겠습니다 (•ㅁ•) ,,,,,!